test/README.md

```diff
@@ -0,0 +1,75 @@
+# Bristlemouth Common Unit Tests
```

Comment:
Love this. Is it worth throwing in integration/contributing docs? @victorsowa12  @towynlin, curious if it would live nicely there on the landing page.

Reply:
Moving it to `docs/reference/unit_tests.md` and have it be built with our documentation.

Reply:
Ah, I see — I do prefer having it here in a readme in the repo because I think curious developers stumble upon it more easily. I could see adding a less technical, more general section in the "API docs" (how I'll refer to the stuff in the docs folder) that points people here. That's my preference, and I'm curious what you all think as well.